ada/rootfs/etc/services.d/ada/finish

```diff
@@ -1,8 +1,12 @@
-#!/usr/bin/execlineb -S0
+#!/usr/bin/env bashio
 # ==============================================================================
-# Take down the S6 supervision tree when Ada fails
+# Take down the S6 supervision tree when service fails
+# s6-overlay docs: https://github.com/just-containers/s6-overlay
 # ==============================================================================
-if { s6-test ${1} -ne 0 }
-if { s6-test ${1} -ne 256 }
 
-s6-svscanctl -t /var/run/s6/services
\ No newline at end of file
+if [[ "$1" -ne 0 ]] && [[ "$1" -ne 256 ]]; then
+  bashio::log.warning "Halt add-on"
+  /run/s6/basedir/bin/halt
```

Comment:
Otherwise it will continue the the restarting message 

```suggestion
  exec /run/s6/basedir/bin/halt
```

Reply:
Oh I might have to fix others then. I just copied what we did in [the example addon](https://github.com/home-assistant/addons-example/blob/main/example/rootfs/etc/services.d/example/finish). We also maybe should update the [blog post](https://developers.home-assistant.io/blog/2022/05/12/s6-overlay-base-images#minimum) again @pvizeli .